internal/prompter/prompter.go

```diff
@@ -13,7 +13,7 @@ import (
 //go:generate moq -rm -out prompter_mock.go . Prompter
 type Prompter interface {
 	Select(string, string, []string) (int, error)
-	MultiSelect(string, string, []string) (int, error)
+	MultiSelect(string, string, []string) ([]string, error)
```

Comment:
For symmetry with Select, would we return `[]int` here?

Reply:
i'm torn about it. no code is actually relying on the new MultiSelect so i haven't played with it (hence the bug) to see how cumbersome a list of ints is to work with. i'll make a call when i'm actually porting a MultiSelect which will be soon.